multichain-testing/scripts/ymax-tool.ts

```diff
@@ -399,10 +428,15 @@ const main = async (
       await walletKit.readPublished('agoricNames.vbankAsset'),
     );
 
+    const networkConfig = getNetworkConfig(env.AGORIC_NET);
```

Comment:
```suggestion
    const { axelarConfig, gmpAddresses } = getNetworkConfig(env.AGORIC_NET)
```

And use the destructured values when passing it to `overridesForEthChainInfo`